documentation/src/main/asciidoc/topics/con_unmarshalled_object_encoding.adoc

```diff
@@ -0,0 +1,9 @@
+[id='unmarshalled-object-encoding_{context}']
+= Plain Old Java Objects (POJO)
+
+For best performance, {brandname} recommends storing unmarshalled POJOs in embedded caches only.
+However, you can configure keys and values with the following media type:
+
+* `application/x-java-object`
+
+//-
```

Comment:
no, maybe it's not strictly necessary but I typically add a comment to make sure any unordered list at the end of an asciidoc file is correctly terminated. sometimes this can lead to asciidoctor "chaining" things together and you get errors with the resulting html. if that makes any sense.